shell/components/ChartPsp.vue

```diff
@@ -0,0 +1,37 @@
+<script>
+import { Checkbox } from '@components/Form/Checkbox';
+import { mapGetters } from 'vuex';
+
+export default {
+  components: { Checkbox },
+  props:      {
+    value: {
+      type:    Object,
+      default: () => {
+        return {};
+      }
+    },
+    mode: {
+      type:    String,
+      default: 'edit'
+    }
+  },
+  created() {
+    if (!this.value.global.cattle) {
+      this.$set(this.value.global, 'cattle', { psp: { enabled: false } });
+    }
+    if (!this.value.global.cattle.psp) {
+      this.$set(this.value.global, 'psp', { enabled: false });
+    }
```

Comment:
That should be setting `global.cattle.psp` instead of `global.psp` - thank you for catching that. Fixing this resolves the error you pointed out below. I've updated the tests to catch this as well.